bluemix/index.js

```diff
@@ -76,20 +71,47 @@ module.exports = yeoman.Base.extend({
         throw new Error('Invalid LoopBack directory');
       }
     }
-  },
-
-  loginToBluemix: function() {
-    bluemix.login.apply(this);
-  },
-
-  promptServiceName: bluemix.promptServiceName,
-  getServicePlans: bluemix.getServicePlans,
-  promptServicePlan: bluemix.promptServicePlan,
-  provisionService: bluemix.provisionService,
-
-  configurePrompt: bluemix.configurePrompt,
-  promptBluemixSettings: bluemix.promptSettings,
-  generateBluemixFiles: bluemix.generateFiles,
-  promptDefaultServices: bluemix.promptDefaultServices,
-  addDefaultServices: bluemix.addDefaultServices,
-});
+  };
+
+  loginToBluemix() {
+    this.login.apply(this);
+  };
+
+  BMpromptServiceName() {
```

Comment:
nitpick: would it be better to have the function name as `BMPromptServiceName` instead of `BMpromptServiceName`? 
Same for the functions below.

Reply:
good catch.